### PR TITLE
feat(unlock-protocol-com) - new website

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description

New website update that will replace the current landing page

# Desktop 

**Screen 1**

Screenshot 2023-06-20 at 06 23 23

**Screen 2**
Screenshot 2023-06-19 at 20 28 20

**Screen 3**
Screenshot 2023-06-19 at 20 28 13

**Screen 4**
Screenshot 2023-06-19 at 20 28 05

**Screen 5**

Screenshot 2023-06-19 at 20 34 25

# Mobile

**Screen 1**
Screenshot 2023-06-19 at 20 27 45

**Screen 2**
Screenshot 2023-06-19 at 20 27 45

**Screen 3**
Screenshot 2023-06-19 at 20 27 45

**Screen 4**
Screenshot 2023-06-19 at 20 27 45

**Screen 5**
Screenshot 2023-06-19 at 20 27 45

**Screen 6**
Screenshot 2023-06-19 at 20 27 45

**Screen 7**
Screenshot 2023-06-19 at 20 27 45

**Screen 8**
Screenshot 2023-06-19 at 20 27 45

# Issues

Fixes #
Refs #

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [x] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Review by julien51 — COMMENTED** [review]
Codewise I have nothing to complain about :)

**Comment by kalidiagne** [comment]
> Codewise I have nothing to complain about :)

Just one thing to add! the correct images on the background

**Comment by kalidiagne** [comment]
Illustration in background 

**Mobile**
Screenshot 2023-06-20 at 16 54 23

**Desktop**
Screenshot 2023-06-20 at 16 50 57

cc: @julien51

**Review by ccarfi — APPROVED** [review]

File: unlock-protocol-com/src/components/pages/Home/sections/HeroSection.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          <span className="text-4xl font-bold text-black md:text-7xl">
+            {title}
+          </span>
+        </div>
+        <div className="flex flex-col items-start gap-8">
+          <div className="text-2xl">{description}</div>
+          <Button
+            as={Link}
+            className="w-auto"
+            href={`${unlockConfig.appURL}/locks/create`}
```
[the remark this answers is not in the document]
Comment (kalidiagne) [reply unlock-protocol-com/src/components/pages/Home/sections/HeroSection.tsx]: already updated

**Comment by kalidiagne** [comment]
Fix images alignment for partners from margin 

**Mobile**
Screenshot 2023-06-20 at 19 29 14

**Desktop**
Screenshot 2023-06-20 at 19 28 57

**Pull request merged**